Makefile

```diff
@@ -37,50 +21,21 @@ endif
 
 default: build
 
-$(GOPATH)/bin/godeps:
-	# godeps needs to be fetched with the insecure flag as launchpad
-	# uses http for part of the checkout process.
-	go get -v -insecure launchpad.net/godeps
-
-# Start of GOPATH-dependent targets. Some targets only make sense -
-# and will only work - when this tree is found on the GOPATH.
-ifeq ($(CURDIR),$(PROJECT_DIR))
-
 build: $(VERSIONDEPS)
-	go build $(PROJECT)/...
```

Comment:
Are these being removed b/c they're not needed w/ the lxd script, or b/c they're just not necessary anymore?

Reply:
Just not necessary anymore.